src/components/qcResultsUpload/QcResultsUploadForm.vue

```diff
@@ -1,54 +1,64 @@
 <template>
-  <div>
-    <traction-form class="text-left" @submit="onSubmit">
-      <traction-sub-section title="Which QC Results would you like to upload?" class="py-6">
-        <traction-select
-          id="used-by-select-input"
-          v-model="usedBySelected"
-          :options="usedByOptions"
-          :state="!!usedBySelected ? true : null"
-          required
-        ></traction-select>
-      </traction-sub-section>
-      <traction-sub-section title="CSV File" class="py-6">
-        <traction-file
-          id="qc-results-upload-file"
-          v-model="file"
-          :state="!!file ? true : null"
-          placeholder="Choose a file or drop it here..."
-          drop-placeholder="Drop file here (CSV only)..."
-          accept="text/csv, .csv"
-          required
-        ></traction-file>
-      </traction-sub-section>
-      <traction-button
-        id="upload-button"
-        type="submit"
-        theme="create"
-        size="lg"
-        :disabled="disableUpload"
-      >
-        <!-- Weird bug - spinner won't show unless button text is two words/ big enough? -->
-        Upload File
-        <traction-spinner v-show="busy"></traction-spinner>
-      </traction-button>
-      <traction-button
-        id="reenable-button"
-        size="lg"
-        :disabled="!disableUpload"
-        @click="disableUpload = !disableUpload"
-        >Re-enable</traction-button
+  <div class="w-3/5 mx-auto bg-gray-100 border border-gray-200 bg-gray-100 rounded-md p-4">
+    <traction-form @submit="onSubmit">
+      <traction-heading level="3" show-border
+        >Which QC Results would you like to upload?</traction-heading
       >
+      <traction-select
+        class="my-5"
+        id="used-by-select-input"
+        v-model="usedBySelected"
+        :options="usedByOptions"
+        :state="!!usedBySelected ? true : null"
+        required
+      ></traction-select>
+      <traction-heading level="3" show-border>CSV File</traction-heading>
+      <traction-file
+        class="my-5 text-left"
+        id="qc-results-upload-file"
+        v-model="file"
+        :state="!!file ? true : null"
+        placeholder="Choose a file or drop it here..."
+        drop-placeholder="Drop file here (CSV only)..."
+        accept="text/csv, .csv"
+        required
+      ></traction-file>
+      <div class="space-x-4 pb-4 flex flex-row justify-end">
+        <traction-button
+          id="upload-button"
+          type="submit"
+          theme="create"
+          size="lg"
+          :disabled="disableUpload"
+        >
+          <!-- Weird bug - spinner won't show unless button text is two words/ big enough? -->
+          Upload File
+          <UploadIcon class="pl-1" />
+          <traction-spinner v-show="busy"></traction-spinner>
```

Comment:
Its not really an issue but I noticed the spinner does cover the UploadIcon when active. Maybe have the spinner replace the UploadIcon conditionally or might be nice to replace the text and icon conditionally.